strategies/RFPStrategy/Milestone/spec.ts

```diff
@@ -34,38 +42,36 @@ class RFPMilestone extends LiveObject {
 
     @BeforeAll()
     setCommonProperties(event: Event) {
-        this.strategyId = event.origin.contractAddress;
+        this.strategy = event.origin.contractAddress;
         this.milestoneId = event.data.milestoneId;
     }
 
     @OnEvent('allov2.RFPSimpleStrategy.MilestonesSet', { autoSave: false })
     @OnEvent('allov2.RFPCommitteeStrategy.MilestonesSet', { autoSave: false })
-    async onMilestonesSet() {
-
+    async onMilestonesSet(event: Event) {
        await this._softDeleteExistingMilestones();
-
         const milestoneLength = event.data.milestonesLength
-        
         const recipientId = await this.contract.acceptedRecipientId()
-
+        const poolId = await this.contract.getPoolId()
         // deno-lint-ignore no-explicit-any
         const rfpMilestones: any = [];
 
         for(let i = 0; i < milestoneLength; i++) {
             const milestone = await this.contract.getMilestone(i)
-            
             const [protocol, pointer] = milestone.metadata
```

Comment:
working to auto-reconstruct the tuple responses here so lemme hit you back on this one

Reply:
ack ! added a TODO here and merging this in